frappe/core/doctype/user/user.py

```diff
@@ -120,11 +120,17 @@ def check_enable_disable(self):
 
 		if not cint(self.enabled):
 			self.a_system_manager_should_exist()
+			# disable notifications if the user has been disabled
+			enable_disable_notifications(self.name, False)
```

Comment:
Minor change, but if you're using immutable objects (booleans, strings, ints, etc.) as arguments, prefer adding the argument name as well.

So:

```suggestion
			enable_disable_notifications(self.name, enabled=False)
```